baseplate/lib/config.py

```diff
@@ -379,6 +389,30 @@ def tuple_of(text: str) -> Sequence[T]:
     return tuple_of
 
 
+def DefaultFromEnv(
+    item_parser: Callable[[str], T], default_src: str
+) -> Callable[[str], OptionalType[T]]:  # noqa: D401
+    """An option of type T or a default.
+
+    The default is sourced from an environment variable with the name ``default_src``.
```

Comment:
```suggestion
    The default is sourced from an environment variable with the name specified in ``default_src``.
```

(just so people don't think the actual name of the env var is "default_src")